src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1445,7 +1445,7 @@ public BundleContext getBundleContext()
      * {@inheritDoc}
      */
     @Override
-    public DtlsControlImpl getDtlsControl(Channel channel)
+    public DtlsControlImpl getSrtpControl(Channel channel)
```

Comment:
If it did we would have to cast. IceUdpTransportManager will still use Dtls, another type of SrtpControl is used in OctoTransportManager (not included in this PR)